src/openads/infrastructure/connector/appnexus/AppNexusConnectorImpl.js

```diff
@@ -46,11 +47,19 @@ export default class AppNexusConnectorImpl extends AppNexusConnector {
   }
 
   loadTags () {
-    this._logger.debug('Loading AppNexus Tags')
```

Comment:
It would be nice to mantain here a trace this._logger.debug('loadTags has been requested') also if we are not calling it, so we will be able to know why and when the loadTags is requested and we can know in the case that 2 loadTags are called, if we need to increase the timeout, and the amount to increase.

Also, can you change:
this._logger.debug('AppNexusConnectorImpl - debounce called')
For
this._logger.debug('loadTags is called')